modules/lib/lib_gui.py

```diff
@@ -0,0 +1,79 @@
+import tkinter as tk
+import pyperclip
+from tkinter import messagebox
+from lib_pass_gen import PasswordGenerator
+
+# This class allows us to create entries with placeholders so it can provide more detail to use wihout additional labels.
+class EntryWithPlaceholder(tk.Entry):
+    def __init__(self, master=None, placeholder="PLACEHOLDER", textvariable=None, color="grey"):
```

Comment:
Is `textvariable` a thing from tkinter? I'm not sure if I like this name. But if it is, then it's fine

Reply:
yes, you can add textvariable to tkinter widget. It allows us to for example link entry value to some varable that may be used later